src/server/server.ts

```diff
@@ -171,11 +171,7 @@ type RouteFunc = (req: Request) => Promise<{ data: any; status: number }>;
 const handleRequest = async (req: Request, res: Response, next: NextFunction, route: RouteFunc) => {
   try {
     const { data, status } = await route(req);
-    if (status === INTERNAL_SERVER_ERROR) {
-      await sendInternalServerError(req, res);
-    } else {
-      sendResponse(res, data, status);
-    }
```

Comment:
Jeg klarer ikke se for meg hvorfor vi ville gjøre dette, men det var sikkert her av en grunn.
Noen som har noen tanker?

Jeg ser jo for meg at vi vil kunne rendre hva vi vil selvom vi skal returnere 500 :thinking:

Reply:
Her vil tro (/håpe) at @gunnarvelle har noen smarte innspill

Reply:
Kan være etterlevninger etter at vi henta m2m token fra auth0 i frontend og måtte håndtere henting av token som feila. Kan sikkert berre rendre errorside sjølv om vi returnerer 500.